src/index.html

```diff
@@ -4,11 +4,404 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Title</title>
+    <link rel="shortcut icon" href="./images/iconn.ico" type="image/x-icon">
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600&display=swap" rel="stylesheet">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+    <header class="header">
+      <div class="menu menu--closed">
+        <div class="burger">
+          <div class="burger__item burger__item_top"></div>
+          <div class="burger__item burger__item_middle"></div>
+          <div class="burger__item burger__item_bottom"></div>
+        </div>
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a href="#" class="menu__link">Home</a>
+          </li>
+          <li class="menu__item">
+            <a href="#about" class="menu__link">About</a>
+          </li>
+          <li class="menu__item">
+            <a href="#skills" class="menu__link">Skills</a>
+          </li>
+          <li class="menu__item">
+            <a href="#portfolio" class="menu__link">Portfolio</a>
+          </li>
+          <li class="menu__item">
+            <a href="#team" class="menu__link">Our team</a>
+          </li>
+          <li class="menu__item">
+            <a href="#contact" class="menu__link">Contact</a>
+          </li>
+          <li class="menu__item">
+            <a href="#signup" class="menu__link">Sign up</a>
+          </li>
+        </ul>
+      </div>
+      <div class="header__navigation navigation">
+        <div class="navigation__wrapper">
+          <a href="#" class="header__logo"></a>
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a href="#about" class="navigation__link">About</a>
+          </li>
+          <li class="navigation__item">
+            <a href="#skills" class="navigation__link">Skills</a>
+          </li>
+          <li class="navigation__item">
+            <a href="#portfolio" class="navigation__link">Portfolio</a>
+          </li>
+          <li class="navigation__item">
+            <a href="#team" class="navigation__link">Our team</a>
+          </li>
+          <li class="navigation__item">
+            <a href="#contact" class="navigation__link">Contact</a>
+          </li>
+        </ul>
+        <a href="#" class="header__button">Sign up</a>
+        </div>
+      </div>
+    </header>
+    <main class="main">
+      <section class="about" id="about">
+        <div class="about__wrapper">
+          <h2 class="about__heading heading">About us</h2>
+          <img
+            src="./images/about/seprater.png"
+            alt="Decorative element"
+            class="about__decoration decoration">
```

Comment:
закрывающую "скобку" также переноси
```suggestion
            class="about__decoration decoration"
          >
```